packages/create/src/actions/create-project.ts

```diff
@@ -35,7 +36,13 @@ export const createProject = async (ctx: Context) => {
 
   spin.start(`Downloading template and creating files...`);
 
-  const defaultTemplate = templates.nextjs;
+  const templatesLookup: Record<Context['framework'], string> = {
+    'Next.js': templates.nextjs,
+    Astro: templates.astro,
+  };
+
+  const defaultTemplate = templatesLookup[ctx.framework];
```

Comment:
Since we have more than one template now, can we rename this variable to just `template`? Github won't let me suggest a rename here that also updates where it is used. 

`const template = templatesLookup[ctx.framework];`

Reply:
Yeah, good call — I was getting confused as per this name — your suggestion makes total sense!